pkg/utils/utils.go

```diff
@@ -124,3 +124,15 @@ func GetChrootExtension() string {
 	}
 	return fmt.Sprintf("chroot %s%s", vars.FilesystemRoot, consts.Host)
 }
+
+func GetPciAddressFromInterfaceName(interfaceName string) (string, error) {
```

Comment:
This function should be part of `NetworkInterface` interface of the host package https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/d9bbbb6d4ccba4e3a750a6c36d05af239fcd4552/pkg/host/types/interfaces.go#L79

+please add docstring